internal/services/dashboard/dashboard_grafana_resource.go

```diff
@@ -105,7 +105,7 @@ func (r DashboardGrafanaResource) Arguments() map[string]*pluginsdk.Schema {
 			},
 		},
 
-		"identity": commonschema.SystemAssignedIdentityOptionalForceNew(),
+		"identity": commonschema.SystemOrUserAssignedIdentityRequiredForceNew(),
```

Comment:
This is a breaking change, did you mean to use:
```suggestion
		"identity": commonschema.SystemOrUserAssignedIdentityOptionalForceNew(),
```

Reply:
Right, that's exactly what I meant. Thanks!